pages/_app.tsx

```diff
@@ -0,0 +1,8 @@
+import "../styles/globals.css";
+import type { AppProps } from "next/app";
+
+function MyApp({ Component, pageProps }: AppProps) {
```

Comment:
For my own understanding, what is this meant for?

Reply:
This was from boilerplate Next v12, not me o.o
Thus why the `MyApp` component takes a `next/app` typed props object. Beyond that, I am not really well versed in what is going on here.